resources/configurations/examples/wan_ds_client.yaml

```diff
@@ -46,7 +46,7 @@ participants:
       - ip: 2.2.2.2
         port: 11670                                               # 13
         external-port: 11680                                      # 14
-        transport: udp                                            # 15
+        transport: tcp                                            # 15
```

Comment:
This needs to be changed I believe. In WAN, having a listening and connection addresses made sense for udp, not with tcp. Also, the connection address is not pointing at the right port (and transport is not udp).